nusight2/src/client/components/localisation/robot_model.ts

```diff
@@ -120,6 +120,7 @@ export class LocalisationRobotModel {
   @observable Rwt: Quaternion; // Torso to world rotation.
   @observable motors: ServoMotorSet;
   @observable fieldLinesDots: { rPWw: Vector3[] };
+  @observable ball: Vector3;
```

Comment:
Can we match field lines so that there is a proper name

```suggestion
  @observable ball: { rBFf: Vector3 };
```

Reply:
The localisation ball is in world space right?

Reply:
I'm just going off `network.ts` which has `robot.ball = Vector3.from(ball.rBWw).applyMatrix4(robot.Hfw);`

So isn't that `rBFf = Hfw * rBWw`?

Reply:
Good point! This should be applied in real-time not in network!